src/ContainerFactory.ts

```diff
@@ -59,6 +60,9 @@ export function createContainer(): Container {
     container.bind(AppealStorageService).toConstantValue(
         new AppealStorageService(getEnvOrThrow(`APPEALS_API_URL`)));
 
+    container.bind(EvidenceUploadService).toConstantValue(
+        new EvidenceUploadService(getEnvOrThrow(`FILE_TRANSFER_API_HOST`), getEnvOrThrow(`FILE_TRANSFER_API_KEY`)));
```

Comment:
A `FILE_TRANSFER_API_HOST` is never used by CH. Please reuse `FILE_TRANSFER_API_URL` which contains protocol information.